.github/workflows/build-and-publish.yml

```diff
@@ -63,7 +67,7 @@ jobs:
       - name: Check if image is already published
         id: check-image-published
         run: |
-          is_image_published=$(./bin/is-image-published "${{ inputs.app_name }}" "${{ inputs.ref }}")
+          is_image_published=$(./bin/is-image-published "${{ inputs.app_name }}" "${{ needs.get-commit-hash.outputs.commit_hash }}")
```

Comment:
Thank you for finding and fixing this bug! I could have sworn I fixed this issue but I was still seeing it happen but didn't have time to look into it